packages/functional-tests/tests/syncV3/signUp.spec.ts

```diff
@@ -39,7 +39,7 @@ test.describe('severity-1 #smoke', () => {
       await login.setPassword(PASSWORD);
       await login.confirmPassword(incorrectPassword);
       await login.setAge('21');
-      await signinTokenCode.clickSubmitButton();
+      await signinTokenCode.submit.click();
```

Comment:
This is an example of the path check probably failing silently. We should be on the signup page here, not `signinTokenCode`. (Same applies for L33 and other sync V3 tests)

Reply:
This has been fixed! I didn't catch it cause I didn't realize these tests were skipped. I verified locally the fix locally now.